app/src/main/contrib/components/Responsibilities/Expectations/ExpectationsDescription.tsx

```diff
@@ -1,15 +1,68 @@
 import * as React from "react";
-import {ExpectationMapping} from "../../../../shared/models/Generated";
+import {Country, ExpectationMapping, NumberRange} from "../../../../shared/models/Generated";
+import {Popover, PopoverBody} from "reactstrap";
+
+interface State {
+    showCountries: boolean;
+}
```

Comment:
I don't having interfaces just called `Props` and `State`, so long as they are in a file with just one component. It gets a bit confusing here. I would suggest you move `CountriesList` and its bits to another file.